storybookjs/storybook
# Angular: Correctly destroy angular application between each render

Issue: noop

## What I did


Before angular it was not destroyed properly. So every NgOnDestrioy method that allows to turn off asynchronous tasks was not called.

Now at each new angular rendering the previous application is correctly destroyed.


## How to test

- Is this testable with Jest or Chromatic screenshots? ✅
- Does this need a new example in the kitchen sink apps? ✅
- Does this need an update to the documentation? ❎



![image](https://media.giphy.com/media/VeSvZhPrqgZxx2KpOA/giphy.gif)

_In my company I am hunting for bad practices that generate memory leaks on Angular. If my colleagues see that I have left this :whip:_
